install_files/securedrop-app-code/DEBIAN/preinst

```diff
@@ -13,12 +13,46 @@ set -e
 # for details, see http://www.debian.org/doc/debian-policy/ or
 # the debian-policy package
 
+
+function permanently_disable_swap() {
+    # Swap usage is prohibited in the context of SecureDrop, due to risk of
+    # forensic discovery recovering plaintext submissions that were written
+    # to disk prior to encrypting. If a swapfile is active, disable it,
+    # and disable all swap entries from /etc/fstab, to prevent mounting
+    # after reboots.
+
+    # Check if swapfile is currently enabled.
+    swap_result=$(swapon --summary)
```

Comment:
Nit: "Declare function-specific variables with `local`. Declaration and assignment should be on different lines." Also: "Always quote... command substitutions...."
```
local swap_result
swap_result="$(swapon --summary)"
```